symengine/lib/symengine_wrapper.pyx

```diff
@@ -1684,7 +1752,11 @@ class NegativeInfinity(Number):
         return True
 
     def __new__(cls):
-        return -oo
+        cdef Basic r = Number.__new__(Infinity)
+        cdef Number s = Number.__new__(Integer)
+        s.thisptr = <RCP[const symengine.Basic]>symengine.integer(-1)
+        r.thisptr = symengine.make_rcp_Infty(s.thisptr)
+        return r
```

Comment:
Currently we return `NegativeInfinity` as `-oo` which uses `c2py(symengine.neg)`. So I wanted to create an instance here. I'm getting this error:
```

Error compiling Cython file:
------------------------------------------------------------
...

    def __new__(cls):
        cdef Basic r = Number.__new__(NegativeInfinity)
        cdef Basic s = Number.__new__(Integer)
        s.thisptr = <RCP[const symengine.Basic]>symengine.integer(-1)
        r.thisptr = symengine.make_rcp_Infty(s.thisptr)
                                             ^
------------------------------------------------------------

symengine_wrapper.pyx:1758:46: Cannot assign type 'RCP[const Basic]' to 'RCP[const Number]'
symengine/lib/CMakeFiles/symengine_wrapper.dir/build.make:63: recipe for target 'symengine/lib/symengine_wrapper.cpp' failed
make[2]: *** [symengine/lib/symengine_wrapper.cpp] Error 1
make[2]: *** Deleting file 'symengine/lib/symengine_wrapper.cpp'
CMakeFiles/Makefile2:139: recipe for target 'symengine/lib/CMakeFiles/symengine_wrapper.dir/all' failed
make[1]: *** [symengine/lib/CMakeFiles/symengine_wrapper.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
```